astronomer/providers/amazon/aws/xcom_backends/s3.py

```diff
@@ -0,0 +1,67 @@
+import json
+import os
+import uuid
+from typing import Any, Union
+
+import pandas as pd
+from airflow.models.xcom import BaseXCom
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+
+
+class S3XComBackend(BaseXCom):
+    """
+    Custom XCom persistence class extends base to support various datatypes.
+    To use this XCom Backend, add the environment variable `AIRFLOW__CORE__XCOM_BACKEND`
+    to your environment and set it to
+    `astronomer.providers.amazon.aws.xcom_backends.s3.S3XComBackend`
```

Comment:
No strong opinion but what others think about `astronomer.providers.amazon.aws.xcom_backends.s3.S3XComBackend` is it ok or do we have better suggestions

Reply:
was trying to follow the comment given for GCS .
https://github.com/astronomer/astronomer-providers/pull/618#discussion_r963695141